pkg/services/validator/azure.go

```diff
@@ -186,9 +159,6 @@ func configureAzureRBACRules(c *components.AzurePluginConfig) error {
 	for {
 		log.InfoCLI("Note: Collecting input for rule #%d", ruleIdx+1)
 
-		// Type is determined first because this is likely how a user would think while they're
-		// using the CLI. This causes some repeated code in other functions called from this one
-		// for name and principal, but this is intentional.
 		ruleType, err := prompts.Select("Rule type", rbacRuleTypes)
```

Comment:
Please remove this selection, since we only provide a single option. We can add it back later if/when we introduce further RBAC-related rules.